lib/context.spec.js

```diff
@@ -0,0 +1,106 @@
+const grpc = require('grpc');
+const Context = require('./context');
+const Response = require('./response');
+
+describe('Context:', () => {
+  let call, context;
+  beforeEach(() => {
+    call = {
+      'request': {'id': '1', 'name': 'name'},
+      'metadata': new grpc.Metadata(),
+      'write': (object) => {
+        return object;
+      },
+      'on': (object) => {
+        return object;
+      },
+      'end': (object) => {
+        return object;
+      },
+    };
+    context = new Context(call);
+  });
+
+  describe('constructor', () => {
+    it('should create a Context instance', () => {
+      expect(context instanceof Context).toBeTruthy();
+    });
+
+    it('should create a Context with a call object', () => {
+      expect(context.call).toBe(call);
+    });
+
+    it('should have a send method', () => {
+      expect(context.send).toEqual(jasmine.any(Function));
+    });
+  });
+
+  describe('send()', () => {
+    it('should receive a Response object', () => {
```

Comment:
expectedResponse no debe ir dentro de 'it' debe estar en un 'beforeEach', ademas se usa en varias lugares (codigo duplicado) por ejemplo en la 59, puede ser una constante global